src/components/common/Toggle/Toggle.style.ts

```diff
@@ -0,0 +1,79 @@
+import styled, { css } from 'styled-components';
+
+import { ToggleContainerProps, ToggleSwitchProps } from './Toggle.type';
+
+export const ToggleContainer = styled.label<ToggleContainerProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  user-select: none;
+  width: ${({ $width }) => $width};
+  height: ${({ $height }) => $height};
+`;
+
+export const ToggleSwitch = styled.div<ToggleSwitchProps>`
+  width: 100%;
+  height: 100%;
+
+  border-radius: ${({ $height }) => $height};
+  background-color: ${(props) => props.theme.container_color};
+  transition: all 0.2s ease;
+  box-sizing: border-box;
+  position: relative;
+
+  &:after {
+    content: '';
+    position: relative;
+    left: 0;
+    top: 0;
+    margin: calc(${({ $height }) => $height} * 0.1);
+    display: block;
+    width: calc(${({ $height }) => $height} * 0.8);
+    height: calc(${({ $height }) => $height} * 0.8);
+    border-radius: 50%;
+    background-color: ${(props) =>
+      props.$isColorReverse ? props.theme.symbol_color : 'white'};
+    transition: all 0.2s ease;
+  }
+
+  &:before {
+    ${({ $isContentShow }) => {
+      if ($isContentShow) {
+        return css`
+          content: '비공개';
+        `;
+      }
+    }};
+    font-size: calc(${({ $width }) => $width} * 0.4);
+    position: absolute;
+    right: calc(${({ $height }) => $height} * 0.2);
```

Comment:
P5; 사소한 의견인데 텍스트가 너무 작지 않나해서요ㅎㅎ 
약간 더 가운데로 모으고 텍스트 크기를 늘려주면 어떨까요?
```suggestion
    font-size: calc(${({ $width }) => $width} * 0.6);
    position: absolute;
    right: calc(${({ $height }) => $height} * 0.4);
```

Reply:
오 좋아요!! 
그리고 생각해보니 저희 어차피 1rem 이하로는 글씨 안 사용하기로 했고... 
글씨 크기는 저렇게 유동적으로 쓰기 보다는 범용적으로 저희 프로젝트 내에서 사용되는 웹 1.6rem/ 모바일 1.4rem 사용하는 게 더 조화로워 보일 것 같아서,
font-size는 웹1.6rem/모바일1.4rem 반응형으로 적용했고, 텍스트 위치도 가운데에 위치하도록 값 조정하였습니다!!